tools/runbooks-from-upstream/go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/projectsyn/component-rook-ceph/tools/runbooks-from-upstream
```

Comment:
Added this as it was helpful to initially render runbooks. SHould I remove it or leave under tools?

Reply:
I'd keep it in tools, doesn't really hurt.